bigtable/spark/README.md

```diff
@@ -0,0 +1,327 @@
+
+```
+cbt \
+  -project=$GOOGLE_CLOUD_PROJECT \
+  -instance=$BIGTABLE_INSTANCE \
+  read $BIGTABLE_TABLE
+```
+
+## Run Wordcount with Cloud Bigtable
+
+### Configure Environment
+
+```
+BIGTABLE_SPARK_JAR=target/scala-2.11/bigtable-spark-samples-assembly-0.1.jar
+BIGTABLE_SPARK_CLASS=example.Wordcount
+BIGTABLE_SPARK_WORDCOUNT_TABLE=wordcount-rdd
+BIGTABLE_SPARK_WORDCOUNT_FILE=README.md
+```
+
+### Configure Cloud Bigtable
+
+```
+cbt \
+  -project=$GOOGLE_CLOUD_PROJECT \
+  -instance=$BIGTABLE_INSTANCE \
+  createtable $BIGTABLE_SPARK_WORDCOUNT_TABLE
+```
+
+```
+cbt \
+  -project=$GOOGLE_CLOUD_PROJECT \
+  -instance=$BIGTABLE_INSTANCE \
+  ls
+```
+
+```
+cbt \
+  -project=$GOOGLE_CLOUD_PROJECT \
+  -instance=$BIGTABLE_INSTANCE \
+  createfamily $BIGTABLE_SPARK_WORDCOUNT_TABLE cf
+```
+
+### Submit Wordcount
+
+```
+$SPARK_HOME/bin/spark-submit \
+  --packages org.apache.hbase.connectors.spark:hbase-spark:1.0.0 \
+  --class $BIGTABLE_SPARK_CLASS \
+  $BIGTABLE_SPARK_JAR \
+  $GOOGLE_CLOUD_PROJECT $BIGTABLE_INSTANCE \
+  $BIGTABLE_SPARK_WORDCOUNT_TABLE $BIGTABLE_SPARK_WORDCOUNT_FILE
+```
+
+### Verify
+
+```
+cbt \
+  -project=$GOOGLE_CLOUD_PROJECT \
+  -instance=$BIGTABLE_INSTANCE \
+  read $BIGTABLE_SPARK_WORDCOUNT_TABLE
+```
+
+### Delete Cloud Bigtable Instance
+
+```
+cbt \
+  -project=$GOOGLE_CLOUD_PROJECT \
+  listinstances
+```
+
+```
+cbt \
+  -project=$GOOGLE_CLOUD_PROJECT \
+  deleteinstance $BIGTABLE_INSTANCE
+```
+
+## Submit DataFrameDemo to Cloud Dataproc
+
+This section describes the steps to submit [DataFrameDemo](src/main/scala/example/DataFrameDemo.scala) application to [Google Cloud Dataproc](https://cloud.google.com/dataproc/).
+
+**TIP** Read [Quickstart using the gcloud command-line tool](https://cloud.google.com/dataproc/docs/quickstarts/quickstart-gcloud) that shows how to use the Google Cloud SDK `gcloud` command-line tool to create a Google Cloud Dataproc cluster and more.
+
+### Configure Environment
+
+```
+BIGTABLE_SPARK_PROJECT_ID=your-project-id
+BIGTABLE_SPARK_INSTANCE_ID=your-bigtable-instance-id
+BIGTABLE_SPARK_DATAPROC_CLUSTER=spark-cluster
+BIGTABLE_SPARK_REGION=europe-west4
+BIGTABLE_SPARK_JAR=target/scala-2.11/bigtable-spark-samples-assembly-0.1.jar
+BIGTABLE_SPARK_CLASS=example.DataFrameDemo
+BIGTABLE_SPARK_TABLE=DataFrameDemo
+```
+
+**NOTE** `BIGTABLE_SPARK_REGION` should point to your region. Read [Available regions and zones](https://cloud.google.com/compute/docs/regions-zones#available) in the official documentation.
+
+### Authenticate
+
+Authenticate to a Google Cloud Platform API using service or user accounts.
+Learn about [authenticating to a GCP API](https://cloud.google.com/docs/authentication/) in the Google Cloud documentation.
+
+**NOTE**: In most situations, we recommend [authenticating as a service account](https://cloud.google.com/docs/authentication/production) to a Google Cloud Platform (GCP) API.
+
+```
+GOOGLE_APPLICATION_CREDENTIALS=your-service-account-json
```

Comment:
nit: the path has to be an absolute path
GOOGLE_APPLICATION_CREDENTIALS=/your/service/account.json

Reply:
fixed